bigquery/storage/managedwriter/conversionutils/doc.go

```diff
@@ -0,0 +1,17 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package conversionutils adds functionality related to converting bigquery representations
+// like schema and data type representations.
+package conversionutils
```

Comment:
I think the utils part of this package name can be dropped. I wonder if a name like `protoadapter` or  just `adapt` would be closer to what this package is used for?

```go
adapt.BQSchemaToStorageTableSchema
protoadapter.BQSchemaToStorageTableSchema

// vs
conversionutils.BQSchemaToStorageTableSchema
```

Reply:
The other functionality I'm anticipating adding here is the ability to generate proto descriptors dynamically based on the schema format.  `adapt` or `convert` works for me, the involvement of non-proto resources makes me hesistant of using `proto` in the package name however.  I'll switch to `adapt` for now.